<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResetCredentials</class>
 <widget class="QWidget" name="ResetCredentials">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="email_LE">
   <property name="geometry">
    <rect>
     <x>492</x>
     <y>120</y>
     <width>261</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(105, 118, 132, 255);
color:rgba(255, 255, 255, 230);
padding-bottom:7px;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="email_verified_LA">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>150</y>
     <width>471</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="security_question_LA">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>200</y>
     <width>421</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="security_question_LE">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>230</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(105, 118, 132, 255);
color:rgba(255, 255, 255, 230);
padding-bottom:7px;
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>400</y>
     <width>641</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0, 0, 0,0);</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="reset_password">
    <widget class="QPushButton" name="confirm_password_PB">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>150</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Reset Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="confirm_password_LE">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>251</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(105, 118, 132, 255);
color:rgba(255, 255, 255, 230);
padding-bottom:7px;
</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="new_password_LE">
     <property name="geometry">
      <rect>
       <x>182</x>
       <y>40</y>
       <width>251</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(105, 118, 132, 255);
color:rgba(255, 255, 255, 230);
padding-bottom:7px;
</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="reset_username">
    <widget class="QPushButton" name="confirm_username_PB">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>80</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Reset username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="new_username_LE">
     <property name="geometry">
      <rect>
       <x>182</x>
       <y>30</y>
       <width>251</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(105, 118, 132, 255);
color:rgba(255, 255, 255, 230);
padding-bottom:7px;
</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="reset_username_PB">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>320</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset username</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset_password_PB">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>320</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>-50</y>
     <width>1341</width>
     <height>811</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0.715909, stop:0 rgba(0, 0, 0, 9), stop:0.375 rgba(0, 0, 0, 50), stop:0.835227 rgba(0, 0, 0, 75));
border-radius:20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>289</x>
     <y>30</y>
     <width>681</width>
     <height>611</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0, 0, 0,100);
border-radius:15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="logIn_LA">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>441</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgba(255, 255, 255, 210);</string>
   </property>
   <property name="text">
    <string>Forgotten username or password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_PB">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>370</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../resources/resources.qrc">:/images/BankLogo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>label_2</zorder>
  <zorder>email_LE</zorder>
  <zorder>email_verified_LA</zorder>
  <zorder>security_question_LA</zorder>
  <zorder>security_question_LE</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>reset_username_PB</zorder>
  <zorder>reset_password_PB</zorder>
  <zorder>logIn_LA</zorder>
  <zorder>back_PB</zorder>
  <zorder>label</zorder>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
